src/scripts/h5p-cornell-content.js

```diff
@@ -26,7 +26,7 @@ export default class CornellContent {
     // Create values to fill with
     this.previousState = Util.extend(
       {
-        dateString: new Date().toLocaleDateString(),
+        dateString: '',
```

Comment:
That will prevent the date that the user started working on the exercise in the titlebar when opening the content for the first time (or before a state was stored).